appinfo.json

```diff
@@ -1,70 +1,76 @@
 {
-  "appKeys": {
-    "bgColor": 0,
-    "txtColor": 1,
-    "starColor": 2
-  },
-  "capabilities": [
-    "configurable"
-  ],
-  "companyName": "exe",
-  "longName": "KlK",
-  "projectType": "native",
-  "resources": {
-    "media": [
-      {
-        "file": "images/font_m.png",
-        "name": "FONT_M",
-        "type": "png"
-      },
-      {
-        "file": "images/font.png",
-        "name": "FONT",
-        "type": "png"
-      },
-      {
-        "file": "images/font_m_red.png",
-        "name": "FONT_M_RED",
-        "type": "png",
-        "targetPlatforms": [
-          "basalt"
+    "appKeys": {
+        "bgColor": 0,
+        "dateColor": 3,
+        "datePositionType": 11,
+        "isEnableDate": 5,
+        "isEnableMonth": 6,
+        "isUseAmPm": 7,
+        "isUseFormal": 10,
+        "isUseLunar": 8,
+        "isUsePrefix": 9,
+        "monthColor": 4,
+        "starColor": 1,
+        "timeColor": 2
+    },
+    "capabilities": [
+        "configurable"
+    ],
+    "companyName": "exe",
+    "enableMultiJS": false,
+    "longName": "KlK",
+    "projectType": "native",
+    "resources": {
+        "media": [
```

Comment:
There is not enough memory of Aplite, so I've prepared a small image.
resources/images/font24~aplite.png
resources/images/font36~aplite.png
resources/images/font48~aplite.png